pom.xml

```diff
@@ -40,18 +27,28 @@
         <dependency>
             <groupId>be.cytomine.client</groupId>
             <artifactId>cytomine-java-client</artifactId>
-            <version>1.4.0</version>
+            <version>2.2.21</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-api</artifactId>
-            <version>2.14.0</version>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
```

Comment:
This dependency raises a Snyk vulnerability. Not easy way to only use log4j v2 ?